test/integration/globals/error.js

```diff
@@ -18,6 +18,40 @@ context('Global Error Page', function() {
       .should('not.exist');
   });
 
+  // NOTE: 'legacy' means the workspace name isn't included in the URL
+  specify('404 not found - legacy routes', function() {
+    cy
+      .visit('/404', { isRoot: true });
+
+    cy
+      .get('.error-page')
+      .should('contain', 'Something went wrong.')
+      .and('contain', ' This page doesn\'t exist.');
+
+    cy
+      .get('.error-page')
+      .contains('Back to Your Workspace')
+      .click();
+
+    cy
+      .get('.error-page')
+      .should('not.exist');
+  });
+
+  // NOTE: 'legacy' means the workspace name isn't included in the URL
+  specify('handle legacy routes', function() {
+    cy
+      .visit('/worklist/owned-by', { isRoot: true });
+
+    cy
+      .url()
+      .should('contain', 'one/worklist/owned-by');
+
+    cy
+      .get('.error-page')
+      .should('not.exist');
+  });
+
```

Comment:
I created these to cover this: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/globals/error_app.js#L50-L60.